### PR TITLE
Update node group role tab

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee [hunk 1]
```diff
@@ -16,29 +17,33 @@ NodeTreeView = OrchestraView.extend(
...
-      url: currentView.options.html.links._self_node_tree
-      method: "GET"
-      success: (response) ->
-        currentView.options.nodes = response
+    if typeof @options.response.site != 'undefined'
+        @options.domContainer.html @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/groupTree/nodeTree')
+        currentView = @
         $.ajax
-          url: currentView.options.html.links._role_list_node
+          url: currentView.options.response.links._self_node_tree
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee]: Don't you check that this param exists before sending the request ?

File: BackofficeBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -263,9 +263,11 @@ open_orchestra_backoffice:
             name: Name
             label: Label
             roles: Roles
+            role_helper: Roles page by page are not taken into account if general right is not enabled
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.en.yml]: I think 'page per page' is more appropriate

**Comment by itkg-alavieille** [comment]
@itkg-ngilain translation added

**Comment by itkg-ngilain** [comment]
:+1:

File: BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee (hunk 1 above)
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee]: if there isn't this param, there is an error of development so the script should not wor.

**Comment by Thiblef** [comment]
:+1:

**Pull request merged**